src/olympia/reviewers/migrations/0009_add_subscription_channel.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.16 on 2020-09-30 14:45
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('reviewers', '0008_auto_20200730_1335'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='reviewersubscription',
+            name='channel',
+            field=models.IntegerField(choices=[(1, 'Unlisted'), (2, 'Listed')], default=2),
```

Comment:
We could maybe use a `PositiveSmallIntegerField`

Reply:
3e5a06c513b72cc1f4418dd012d7ce82904e8eea